### ISSUE TITLE
Set up guidelines for Slack Etiquette

### ISSUE THREAD

**Issue by mjang** [writethedocs/www#631]
Candidate: fork from https://github.com/SparkPost/slack-etiquette, recommended by devrelcollective

**Comment by Bradamant3** [comment]
Adding a couple of related ideas here, for tracking:
- include reminders about other WTD resources (newsletter, conferences, meetups, guide)
- post a reminder once a month, maybe in #announcements but with a pointer in #general (I've seen other Slacks do this quite nicely)

**Comment by mjang** [comment]
Another idea, credit @bethaitman in WtD Slack -- from Stack Overflow
<img width="934" alt="screen shot 2018-09-06 at 15 20 18" src="https://user-images.githubusercontent.com/3287976/45166688-74fc8e80-b1ac-11e8-86fb-133fa0774de7.png">

**Comment by mjang** [comment]
One more idea from devrelcollective, link to CoC

![welcome](https://user-images.githubusercontent.com/3287976/45256019-65fd1400-b345-11e8-8d6a-1236c842dc95.png)

**Comment by plaindocs** [comment]
Hey Folks, what is the state of this? Who needs to do what? @Bradamant3 @mjang

**Comment by Bradamant3** [comment]
We talked a while back about automating a post to the _announcements channel on a monthly basis, reminding Slackers of guidelines and CoC. That said, our Slack moderation is still informal, undocumented, unsystematic, and I think we need to address that issue (as part of larger governance issues or separately) before we go making any more noise about guidelines or CoCs. I'm fine to close this issue and open a new one when we have something resembling an agreed-upon and documented system in place. But @mjang should weigh in.

**Comment by mjang** [comment]
One of the unknowns -- the way we enforce slack guidelines goes beyond the CoC, and we don't make that clear. 

And @Bradamant3 is correct -- the "unsystematic" nature of what we do, the lack of communication between some of us, means that we're not consistent in how we enforce requirements, at least on Slack. I know every time I post something even slightly controversial, I fear violating our guidelines or CoC or I'm not sure. 

Bottom line, given the current state of our governance, I agree with:

> I'm fine to close this issue and open a new one when we have something resembling an agreed-upon and documented system in place.

**Comment by mjang** [comment]
cc @nbkaplan  (to complete the loop)

**Comment by plaindocs** [comment]
Also seems fine to leave it open as a reminder as it is an ongoing thing. Whatever works. 

:+1: on clear written slack guidelines we can all point to.

**Comment by MarcelinaH** [comment]
I don't know if it's the right place, but... I think it would be good to set a rule to inform the organisers of the Write the Docs meet-ups on the existence of Slack, so they could use it and pass the information to the meet-up participants. As WTD keeps on growing, local organisers might also have a fresh approach to the etiquette rules.
